charts/okapi/Chart.yaml

```diff
@@ -24,5 +24,5 @@ appVersion: 4.01.0
 
 dependencies:
 - name: common
-  version: 0.1.0
+  version: x.x.x
```

Comment:
Is this is correct version?

Reply:
Locally it works, wanna check with Nexus.